src/devtools/devtools.tsx

```diff
@@ -229,6 +238,7 @@ export function ReactQueryDevtools({
       <ThemeProvider theme={theme}>
         <ReactQueryDevtoolsPanel
           ref={panelRef as any}
+          styleNonce={styleNonce || (window as any).__webpack_nonce__}
```

Comment:
I'm fine with the style nonce, but I'd rather not couple this to window and `__webpack_nonce__`. It should be easily doable in user land to forward this:

```
<ReactQueryDevtools styleNonce={window.__webpack_nonce__}
```

Reply:
removed